src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/e478cc95-f26d-4aa3-ab78-2ef49e630273)

Reply:
I don't fully understand what can be taken outside the method level here